docs/viplab-3.0/computation_template.md

```diff
@@ -326,37 +363,59 @@ Note: `//` with text following until EOL is a comment,
               "metadata" : {
                 "guiType" : "input_field",
                 "type": "number",
-                "name": "time_delay"
+                "name": "Age",
+                "description" : "Enter your current age"
               },
               "default": [10],
               "min": 0,
               "max": 500,
               "step": 0.1,
               "validation": "range"
-            },
+            }
+          ],
+          "content"   : "W2Fib3V0IHlvdV0NCmxpa2VkVGhpbmdzPXt7X19jaGVja2JveF9ffX0NCmZhdm9yaXRlUEw9e3tfX3JhZGlvQnV0dG9uX199fQ0KZnJpZGdlPXt7X19kcm9wZG93blNpbmdsZV9ffX0NCmRhbmNpbmc9e3tfX2Ryb3Bkb3duTXVsdGlwbGVfX319DQpkaXNsaWtlZFRoaW5ncz17e19fdG9nZ2xlX199fQ0KcmFuZG9tTnVtYmVycz17e19fc2xpZGVyTXVsdGlwbGVfX319DQpuYW1lPXt7X19pbnB1dFRleHRXT01heGxlbmd0aF9ffX0NCmNocmlzdG1hc1dpc2g9e3tfX2lucHV0VGV4dFdNYXhsZW5ndGhfX319DQphZ2U9e3tfX2lucHV0TnVtYmVyX199fQ"
```

Comment:
Add a comment, that the encoded can be found below the example

Reply:
wouldn't it be more consistent to add it as a comment underneath, just as is is with the other ones?
I would add it like this:
// decoded:
 // [about you]
 // likedThings={{__checkbox__}}
 // favoritePL={{__radioButton__}}
 // fridge={{__dropdownSingle__}}
 // dancing={{__dropdownMultiple__}}
// dislikedThings={{__toggle__}}
// randomNumbers={{__sliderMultiple__}}
// name={{__inputTextWOMaxlength__}}
// christmasWish={{__inputTextWMaxlength__}}
// age={{__inputNumber__}}

Reply:
The goal is to use this example in the tutorial, where the input is explained more. Maybe we later could reference this: "For more info on input go to..."